tutorials/gradleTutorial.md

```diff
@@ -30,10 +30,10 @@ As a developer, you write a _build file_ that describes the project. A build fil
    git checkout master
    git merge gradle
    ```
-1. Open the `build.gradle` file in an editor. Update the following code block to point to the main class (i.e., the one containing the `main` method) of your application. The code below assumes your main class is `seedu.duke.Duke`
+1. Open the `build.gradle` file in an editor. Update the following code block to point to the main class (i.e., the one containing the `main` method) of your application. The code below assumes your main class is `seedu.duke.duke.Duke`
    ```groovy
    application {
-       mainClassName = "seedu.duke.Duke"
+       mainClassName = "seedu.duke.duke.Duke"
```

Comment:
Oh wow you even changed the tutorial to use the fully qualified name here! But, does your file structure even use seedu.duke.duke?